console.php

```diff
@@ -59,14 +59,16 @@ function exceptionHandler($exception) {
 
 	set_exception_handler('exceptionHandler');
 
+	$config = \OC::$server->getConfig();
+
 	if (!function_exists('posix_getuid')) {
 		echo "The posix extensions are required - see https://www.php.net/manual/en/book.posix.php" . PHP_EOL;
 		exit(1);
 	}
 	$user = posix_getuid();
-	$configUser = fileowner(OC::$configDir . 'config.php');
+	$configUser = fileowner($config->getSystemValue('datadirectory', \OC::$SERVERROOT . '/data'));
```

Comment:
```suggestion
	$dataDirectoryUser = fileowner($config->getSystemValue('datadirectory', \OC::$SERVERROOT . '/data'));
```
Need to next lines too